angular/core/components/thread_page/thread_page.haml

```diff
@@ -5,12 +5,13 @@
     .thread-page__main-content
       %outlet.before-column-right.lmo-column-right{name: "before-thread-page-column-right", ng-if: "threadPage.eventsLoaded", model: "threadPage.discussion"}
       %poll_common_card.lmo-card--no-padding.lmo-column-right{poll: "poll", ng-repeat: "poll in threadPage.discussion.activePolls() track by poll.id"}
-      %decision_tools_card.lmo-column-right{ng-if: "threadPage.canStartPoll()", discussion: "threadPage.discussion"}
+      %decision_tools_card.lmo-column-right{discussion: "threadPage.discussion"}
       %poll_common_index_card.lmo-column-right{model: "threadPage.discussion"}
-      %members_card.lmo-column-right{ng-if: "threadPage.canViewMemberships()", group: "threadPage.discussion.group()"}
+      %members_card.lmo-column-right{group: "threadPage.discussion.group()"}
```

Comment:
In this case we had 2 identical methods defined across 2 controllers both checking if the user could view members to toggle displaying the card. We don't have any other conditions underwhich we do don't display. 

It made sense to be because I don't think the threadPageController needs to know under what conditions someone can see the membership of a group.